### DISCUSSION TITLE
Idea for final thesis project

### DISCUSSION THREAD

**Discussion by CROprogrammer** [MarimerLLC/csla#3266] [General]
Hi,
I'm totally new in the .NET and CSLA .NET World and for my final thesis I have to demonstrate CSLA .NET framework's capabilities. I have to create web and mobile apps with identical functionalities using CSLA .NET, other frameworks (for UI, Data access, etc.) are not predetermined.
What projects would be appropriate to demonstrate full (or at least most of) CSLA .NET's capabilities?
I need something more complex than project tracker.

Thank you in advance :)

**Comment by Chicagoan2016** [comment]
>  I need something more complex than project tracker.
> 
You could develop something of your own (based on Csla) but how much time do you have?
We all have developed applications (Csla based) at our work.

**Reply by CROprogrammer** [reply]
Yeah I need to develop something on my own to demonstrate CSLA's capabilities. Let's say that I have about 40 days to finish my work. And I'm currently searching for project idea which will best demonstrate CSLA's capabilities. 
Do you have any project idea using which I will be able to do so?

**Reply by Chicagoan2016** [reply]
@michelh-fer you mentioned you are 'totally' new to the .NET and Csla world, in that case, I would extend the ProjectTracker project.
I am assuming you are studying Computer Science (or Engineering), you could take one of your projects, preferably resembling a 'business' scenario and port it to Csla.

**Reply by CROprogrammer** [reply]
Thank you for your input, extending ProjectTracker seems like a good idea.
